backend/config/settings.py

```diff
@@ -92,6 +89,7 @@
     "DESCRIPTION": "This is the documentation for the Dr-Trottoir API",
     "VERSION": "1.0.0",
     "SERVE_INCLUDE_SCHEMA": False,
+    # TODO: Ask @Seppe what this line does and if it's still necessary
```

Comment:
I believe this is what make the api routes appear like 
![image](https://user-images.githubusercontent.com/108656952/235351188-51212336-c07c-4bed-a381-91d30a67d5dd.png)
instead of `/authentication...`
But since the api will be moved to a seperate port this because outdated. This could be removed but maybe specify in the description that the api runs on port 2002

Reply:
Alright the prefix has been removed and I added the port 2002 in the description.